chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/PendingCertifiedProductDetails.java

```diff
@@ -353,6 +355,19 @@ public PendingCertifiedProductDetails(PendingCertifiedProductDTO dto) {
                 cert.setTestStandards(null);
             }
 
+            if (certCriterion.getOptionalStandards() != null) {
+                for (PendingCertificationResultOptionalStandardDTO os : certCriterion.getOptionalStandards()) {
+                    CertificationResultOptionalStandard optionalStd = new CertificationResultOptionalStandard();
+                    optionalStd.setId(os.getId());
+                    optionalStd.setCitation(os.getCitation());
+                    optionalStd.setOptionalStandardId(os.getOptionalStandardId());
+                    optionalStd.setOptionalStandard(os.getOptionalStandard());
```

Comment:
Can we use `CertificationResultOptionalStandard.builder()` to create this object?